build-run-markii.sh

```diff
@@ -11,7 +11,9 @@ $DIR/check-jdk-version
 
 APK_PATH=$(realpath $1)
 OUTPUT_PATH=$(realpath $2)
-TEMPFILE_REMOVE= $3
+
+TEMPFILE_REMOVE=$3
```

Comment:
`$3` might be empty. Set it to a default value:

```bash
if [ -z "$3" ]; then
    TEMPFILE_REMOVE=...
else
    TEMPFILE_REMOVE=$3
fi
```

Test it though, I am not sure I wrote it in the right way